waitress/task.py

```diff
@@ -48,6 +44,16 @@
     'upgrade'
 ))
 
+PROXY_HEADERS = frozenset({
+    'X_FORWARDED_FOR',
+    'X_FORWARDED_HOST',
+    'X_FORWARDED_PROTO',
+    'X_FORWARDED_PORT',
+    'X_FORWARDED_BY',
+    'FORWARDED',
+})
```

Comment:
it seems odd that this set is defined both above and here, I'd suggest coming up with a way to define it in one spot and to normalize on the format that's most performance-sensitive